MAUI/Calendar/overview.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: About .NET MAUI Calendar control | Syncfusion
+description: Learn here the overview of Syncfusion .NET MAUI Calendar(SfCalendar) control, its basic features and functionalities of calendar.
+platform: MAUI
+control: SfCalendar
+documentation: ug
+---
+
+# Overview of .NET MAUI Calendar (SfCalendar)
+
+The Syncfusion .NET MAUI Calendar (SfCalendar) is a lightweight calendar control that allows user to easily select a Single, Multiple and a Range of dates. It offers Month, Year, Decade, and Century view options to quickly navigate to the desired date. To restrict the date selection by using the minimum, maximum, enable past dates, and disabled dates.
```

Comment:
Check and remove the lightweight

Reply:
Checked and removed the lightweight word